views.md

```diff
@@ -86,11 +86,11 @@ Occasionally, you may need to share a piece of data with all views that are rend
     }
 
 <a name="view-composers"></a>
-## View Composers
+## 视图组件
 
-View composers are callbacks or class methods that are called when a view is rendered. If you have data that you want to be bound to a view each time that view is rendered, a view composer can help you organize that logic into a single location.
+视图组件是在视图渲染时调用的一些回调或者类方法。如果你需要在某些视图渲染时绑定一些数据上去，那么视图组件就是你的的不二之选，另外他还可以帮你将这些绑定逻辑整理到特定的位置。
 
-For this example, let's register the view composers within a [service provider](/docs/{{version}}/providers). We'll use the `View` facade to access the underlying `Illuminate\Contracts\View\Factory` contract implementation. Remember, Laravel does not include a default directory for view composers. You are free to organize them however you wish. For example, you could create an `App\Http\ViewComposers` directory:
+下面例子中，我们会在一个 [服务提供者](/docs/{{version}}/providers) 中注册一些视图组件。同时使用视图 Facade 来取用对底层 `Illuminate\Contracts\View\Factory` 的实现。注意：Laravel 没有存放视图组件的默认目录，但你可以根据自己的喜好来重新组织，例如：`App\Http\ViewComposers`。
 
```

Comment:
了解
CharlieJade <notifications@github.com>于2016年9月8日 周四23:02写道：

> In views.md
> <https://github.com/laravel-china/laravel-docs/pull/6#discussion_r78024276>
> :
>
> >
> > -For this example, let's register the view composers within a [service provider](/docs/{{version}}/providers). We'll use the `View` facade to access the underlying `Illuminate\Contracts\View\Factory` contract implementation. Remember, Laravel does not include a default directory for view composers. You are free to organize them however you wish. For example, you could create an `App\Http\ViewComposers` directory:
> > +下面例子中，我们会在一个 [服务提供者](/docs/{{version}}/providers) 中注册一些视图组件。同时使用视图 Facade 来取用对底层 `Illuminate\Contracts\View\Factory` 的实现。注意：Laravel 没有存放视图组件的默认目录，但你可以根据自己的喜好来重新组织，例如：`App\Http\ViewComposers`。
> >
>
> 同时使用视图 Facade 来取用对底层 Illuminate\Contracts\View\Factory 的实现。
>
> 建议修改为
>
> 同时使用 View Facade 来访问 Illuminate\Contracts\View\Factory contract 的底层实现。
>
> 这里需要说一下，文档里当使用 代码加亮 符号的话，如：
>
> `View` 被包起来的话，就算是代码，不应该翻译。
>
> 请把文章里的其他地方做相应修改。
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/laravel-china/laravel-docs/pull/6/files/5a1db299efbfdaa83d0648ce8852bd52dd719a48#r78024276>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABYSO3Q0EyE9Gctt6IAcl1MMUJ_3Yx5sks5qoCNvgaJpZM4J4D4M>
> .
>